app/app.go

```diff
@@ -167,13 +169,15 @@ type RegenApp struct {
 	IBCKeeper        *ibckeeper.Keeper // IBC Keeper must be a pointer in the app, so we can SetRouter on it correctly
 	EvidenceKeeper   evidencekeeper.Keeper
 	TransferKeeper   ibctransferkeeper.Keeper
-	//nolint
-	wasmKeeper wasm.Keeper // only used in experimental builds
+	FeeGrantKeeper   feegrantkeeper.Keeper
+	AuthzKeeper      authzkeeper.Keeper
+
+	// wasmKeeper wasm.Keeper // only used in experimental builds
```

Comment:
I believe we are having single RegenApp definition, so we would need wasmkeeper here inorder to make experimental modules work

Reply:
I'm not sure but I think experimental modules works fine even without wasmkeeper